### PR TITLE
Add apk build date in About Fragment

### LINKED ISSUE #14691: Add a date near the version number.
In order to know whether there was a recent update in the background, it would be nice to have the date of the release near a version number.
I'm running on alpha 3 directly from the play store. I didn't see any update message (just a request to help developers by sending feedback). I thought some things changed but was not sure whether I imagined it or not. The release date would have confirmed that I got a recent version.

### PR DESCRIPTION
**Pull request by oyeraghib**
## Purpose / Description
Add apk build date in About Fragment near version number

## Fixes
* Fixes #14691 

## How Has This Been Tested?
Manually tested on emulator Pixel 3a

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/build.gradle [hunk 1]
```diff
@@ -50,6 +50,7 @@ android {
         buildConfigField "Boolean", "ENABLE_LEAK_CANARY", "false"
         buildConfigField "Boolean", "ALLOW_UNSAFE_MIGRATION", "false"
         buildConfigField "String", "GIT_COMMIT_HASH", "\"${gitCommitHash()}\""
+        buildConfigField "String", "BUILD_TIME", "\"" + System.currentTimeMillis().toString() + "\""
```
Comment (BrayanDSO) [inline AnkiDroid/build.gradle:53]: Are the `"\""` necessary?

**Review by BrayanDSO — COMMENTED** [review]
Maybe put the date below the version?

**Comment by oyeraghib** [comment]
Update with changes (date below version)

File: AnkiDroid/build.gradle (hunk 1 above)
Comment (oyeraghib) [reply AnkiDroid/build.gradle]: removed.

File: AnkiDroid/build.gradle [hunk 2]
```diff
@@ -50,6 +50,7 @@ android {
         buildConfigField "Boolean", "ENABLE_LEAK_CANARY", "false"
         buildConfigField "Boolean", "ALLOW_UNSAFE_MIGRATION", "false"
         buildConfigField "String", "GIT_COMMIT_HASH", "\"${gitCommitHash()}\""
+        buildConfigField "String", "BUILD_TIME", "\"${System.currentTimeMillis().toString()}\""
```
Comment (david-allison) [inline AnkiDroid/build.gradle:53]: I believe this can be a `Long`. This would remove `BuildConfig.BUILD_TIME.toLong()`

**Review by david-allison — CHANGES_REQUESTED** [review]
LGTM, one quick cleanup. 

Consider this a soft approval

**Review by david-allison — APPROVED** [review]
Cheers, I'm happy if tests pass

**Review by lukstbit — APPROVED** [review]
Thanks!

**Comment by Arthur-Milchior** [comment]
Thanks a lot, that’ll be quite helpful!

**Pull request merged**